homework/express-cat/app.js

```diff
@@ -0,0 +1,51 @@
+var express = require('express');
+var path = require('path');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+var exphbs  = require('express-handlebars');
+var mongoose = require('mongoose');
+
+
+var index = require('./routes/index');
+var catroutes = require('./routes/catroutes');
+var models = require('./modelsdb')
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function (callback) {
+    console.log('Yay!')
+    var Cat = models.catModel
```

Comment:
This seems like unused code?
You should not need your model in your `app.js` file.